### PR TITLE
basic enhancement

### PR DESCRIPTION
**Pull request by void-main**
added 3 basic enhancements:
1. Inform user when the installation is failed while installing ruby or rails. Currently, the program will tell the user that installation is successful even when it actually failed.
2. Disable the checkPrerequisitesButton during the installation process. It seems weird to me that I can check the prerequisites when I'm actually installing.
3. Skip ruby installation process when it is already installed.

### CODE REVIEW

File: RailsOneClick/AppController.m [hunk 1]
```diff
@@ -15,13 +15,23 @@ @implementation AppController
 @synthesize view;
 @synthesize currentViewController;
 
-- (BOOL)isRailsInstalled
+- (BOOL)isFileExists:(NSString *)path
```
Comment (oscardelben) [inline RailsOneClick/AppController.m:18]: I would just call this `fileExists:`

**Comment by oscardelben** [comment]
This is good, thanks a lot!

File: RailsOneClick/AppController.m (hunk 1 above)
Comment (void-main) [inline RailsOneClick/AppController.m:18]: sorry for that...

**Comment by void-main** [comment]
glad you like it!

:)

**Pull request merged**